File: Lab_2/src/Kwadrat.java
public class Kwadrat extends Czworokat
{

    Kwadrat(double bok)
    {
        super(bok, bok, bok, bok, 90);
    }

    @Override
    public double count_Pole()
    {
        pole = Math.pow(bok1, 2);
        pole = Math.round(pole * 10.0) / 10.0;
        return pole;
    }

}

File: Lab_2/С++/Kwadrat.cpp
#include "Kwadrat.h"

    Kwadrat::Kwadrat(double bok) : Czworokat (bok, bok, bok, bok, 90) { }

    double Kwadrat::count_Pole()
    {
        pole = pow(bok1, 2);
        pole = round(pole * 10.0) / 10.0;
        return pole;
    }

